robotpy_ext/misc/periodic_filter.py

```diff
@@ -0,0 +1,13 @@
+class PeriodicFilter:
+    def __init__(self, parent):
+        '''
+        :type parent: Object
+        '''
+        
+        self.parent = parent
+        
+    def filter(self, record):
+        if self.parent.loggingLoop:
+            return True
+        else:
+            return record.levelno > 20
```

Comment:
Shouldn't this use the named constant `logging.INFO` then? `20` isn't meaningful to… well, anyone using the `logging` module.